src/components/inspectors/InspectorPanel.vue

```diff
@@ -76,24 +76,38 @@ export default {
         return Process.inspectorConfig;
       }
 
+      const inspectorConfig = cloneDeep(this.nodeRegistry[type].inspectorConfig);
+
       if (this.isSequenceFlow(type) && this.isConnectedToGateway(definition)) {
-        return sequenceExpression;
+        const expressionConfig = {
+          component: 'FormInput',
+          config: {
+            label: 'Expression',
+            helper: 'Enter the expression that describes the workflow condition',
+            name: 'conditionExpression',
+          },
+        };
+
+        inspectorConfig[0].items[1].items.push(expressionConfig);
```

Comment:
For readability, do you think extracting  **inspectorConfig[0].items[1].items** to `const sequenceFlowConfig = inspectorConfig[0].items[1].items`, would be better ?